CONTRIBUTING.md

```diff
@@ -16,20 +16,30 @@ automatic check once you submit a PR, but you can also sign it after opening you
 
 ## Style guideline
 
-We follow the [Google Java Style
-Guide](https://google.github.io/styleguide/javaguide.html). Our build
-automatically will provide warnings for simple style issues.
+We follow the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html). 
+Our build will fail if source code is not formatted according to that style.
 
-Run the following command to format all files. This formatter uses
-[google-java-format](https://github.com/google/google-java-format):
+To verify code style manually run the following command, 
+which uses [google-java-format](https://github.com/google/google-java-format) library:
 
-### OS X or Linux
+`./gradlew verifyGoogleJavaFormat`
 
-`./gradlew goJF`
+or on Windows
 
-### Windows
+`gradlew.bat verifyGoogleJavaFormat`
 
-`gradlew.bat goJF`
+Instead of fixing style inconsistencies by hand, you can run gradle task `googleJavaFormat`
+to automatically fix all found issues:
+
+`./gradlew googleJavaFormat`
```

Comment:
`goJF` shortcut

Reply:
Are you sure about that? I would prefer to have full names in official documentation. Makes discovery easier for newcomers. With abbreviation it is harder to Ctrl+F your way around.

Reply:
I also prefer the full name in the documentation.